cli/print/clusters.go

```diff
@@ -10,14 +10,15 @@ import (
 )
 
 // TODO: implement a -o wide, and expose nodecount, vcpus and memory also?
+
 var clustersTmplSrc = `ID	NAME	K8S DISTRO	K8S VERSION	STATUS	CREATED	EXPIRES
 {{ range . -}}
 {{ .ID }}	{{ .Name }}	{{ .KubernetesDistribution}}	{{ .KubernetesVersion	}}	{{ .Status }}	{{ .CreatedAt}}	{{ .ExpiresAt }}
 {{ end }}`
 
 var clustersTmpl = template.Must(template.New("clusters").Funcs(funcs).Parse(clustersTmplSrc))
 
-func Clusters(outputFormat string, w *tabwriter.Writer, clusters []*types.Cluster) error {
+func Clusters(outputFormat string, w *tabwriter.Writer, clusters []*types.Cluster, includeHeader bool) error {
```

Comment:
Is `includeHeader` still needed?

Reply:
Yeah that was an oversight I didn't think to remove after changing how it printed to the screen. Definitely not needed and I'll take care of that now